generators/guestcontainer/index.js

```diff
@@ -99,6 +104,16 @@ module.exports = generators.Base.extend({
             var serviceName = this.props.serviceName;
             var appTypeName = this.projName + 'Type';
             var instanceCount = this.props.instanceCount;
+            if(this.props.portMap != ""){
+                var portMap = this.props.portMap.split(":");
+                var portMapContainer = portMap[0];
+                var portMapHost = portMap[1];
+            }
+            else{
```

Comment:
you can just avoid this else block if you initialized these two variables above the if block right?

Reply:
Done. thanks